updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,134 @@
+** Mirror the images and configuration manifests to a directory on the removable media.
++
+----
+$ oc adm release mirror "--to-dir=${REMOVABLE_MEDIA_PATH}/mirror" quay.io/openshift-release-dev/ocp-release:4.3.0-x86_64
+----
++
+
+** Take the media to the disconnected environment and upload the images to the local container registry
++
+----
+$ oc image mirror "--from-dir=${REMOVABLE_MEDIA_PATH}/mirror" file://openshift/release:4.3.0* REGISTRY/REPOSITORY
+----
++
+. Use `oc` to log in to a given cluster to be upgraded.
++
+[NOTE]
+====
+If you are upgrading from earlier release than 4.5.0, you must must use the manual method of creating configmap.
+
+This is because `oc` command enhancement is not available in previous releases.
+====
++
+. Apply the the mirrored release image signature ConfigMap to the connected cluster
++
+----
+$ oc apply -f mirror/config/signature-sha256-81154f5c03294534.yaml
+----
+. Perform the upgrade
++
+----
+$ oc adm upgrade --allow-explicit-upgrade --to-image REGISTRY/REPOSITORY@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+----
+
+[id="update-disconnected-manual_method_configmap_creation"]
+== Manual method to create the image signature configmap
+
+The following commands will create and apply the image signature configmap to the cluster you want to update. 
+
+.Procedure
+
+. Find the link:https://access.redhat.com/solutions/4583231[version] you want the cluster to update.
+
+. Add the version to OCP_RELEASE_NUMBER environment variable.
++
+----
+$ export OCP_RELEASE_NUMBER=<4.y.z>
+----
++
+. Get the release image digest from link:https://quay.io/[Quay].
++
+----
+$ export DIGEST="$(oc adm release info quay.io/openshift-release-dev/ocp-release:${OCP_RELEASE_NUMBER}-x86_64 | sed -n 's/Pull From: .*@//p')"
+----
++
+. Get the image signature from link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com] website.
++
+----
+$ export SIGNATURE_BASE64=$(curl -s "https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release/$(echo $DIGEST | cut -d: -f1)=$(echo $DIGEST | cut -d: -f2)/signature-1" | base64 -w0 && echo)
+----
++
+. Get the digest algorithm.
++
+----
+export DIGEST_ALGO=$(echo $DIGEST | cut -d: -f1)
+----
++
+. Get the digest signature.
++
+----
+export DIGEST_SIGNATURE=$(echo $DIGEST | cut -d: -f2)
+----
++
+. Create the configmap.
++
+----
+$ cat >checksum-${OCP_RELEASE_NUMBER}.yaml <<EOF
+ apiVersion: v1
+ kind: ConfigMap
+ metadata:
+   name: release-image-${OCP_RELEASE_NUMBER}
+   namespace: openshift-config-managed
+   labels:
+     release.openshift.io/verification-signatures: ""
+ binaryData:
+   ${DIGEST_ALGO}-${DIGEST_SIGNATURE}: ${SIGNATURE_BASE64}
+ EOF
+----
++
+. Install the config map in the cluster which needs to be updated.
++
+----
+$ oc apply -f checksum-${OCP_RELEASE_NUMBER}.yaml
+----
++
+. Get the created configmap to verify.
+----
+$ oc get cm release-image-${OCP_RELEASE_NUMBER} -o yaml
```

Comment:
Hopefully `oc apply` tells you if you fail.  I don't think we need to suggest folks double-check.